src/provider/SiftScienceProvider.spec.ts

```
@@ -24,64 +23,126 @@ describe("SiftScience Gateway - ", () => {
+      const merchantSettings = {
+        ...merchantSettingsResponse,
+        prodBaconKey: "",
+        sandboxBaconKey: null
+      };
+
+      const data = siftScienceService.createSiftScienceSession(
+        processor,
+        clientIdentification,
+        merchantSettings
+      );
+
+      executeLoadScript();
+
+      await expect(data).resolves.toHaveProperty("userId", undefined);
+      await expect(data).resolves.toHaveProperty("sessionId", undefined);
+    });
+
+    it("should throw error when load script fails", async () => {
+      const data = siftScienceService.createSiftScienceSession(
+        processor,
+        clientIdentification,
+        merchantSettingsResponse
+      );
 
-    expect(data).toHaveProperty("userId");
-    expect(data).toHaveProperty("sessionId");
+      executeErrorScript();
+
+      await expect(data).rejects.toHaveProperty("code", "E022");
+    });
   });
 
-  it("test createSiftScienceSession request - without env", async () => {
-    const merchantSettings = {
-      ...merchantSettingsResponse,
-      prodBaconKey: "",
-      sandboxBaconKey: null
-    };
-
-    const data = siftScienceService.createSiftScienceSession(
-      processor,
-      clientIdentification,
-      mockKushki,
-      merchantSettings
-    );
-
-    expect(data).toHaveProperty("userId", undefined);
-    expect(data).toHaveProperty("sessionId", undefined);
+  describe("createSiftScienceAntiFraudSession - method", () => {
+    const userIdMock = "1234412";
+
+    it("should return sift object when merchant have sift enabled", async () => {
+      const data = siftScienceService.createSiftScienceAntiFraudSession(
+        userIdMock,
+        merchantSettingsResponse
+      );
+
+      executeLoadScript();
+
+      await expect(data).resolves.toHaveProperty("userId");
+      await expect(data).resolves.toHaveProperty("sessionId");
+    });
+
+    it("should throws error when merchant have sift disabled", async () => {
```

aca ponle como 
should throw an error when merchant have sift disabled